src/rust/Cargo.toml

```diff
@@ -58,7 +58,8 @@ itertools = "=0.11.0"
 version = "=22.0.0"
 git = "https://github.com/stellar/rs-soroban-env"
 package = "soroban-env-host"
-rev = "81a4239f3e1ab5e4030140310faa64aba28dfdcb"
+rev = "5ed156b7fb5dd3e46635aa0f818d5db33dfd6b6a"
+#path = "../../../rs-soroban-env/soroban-env-host"
```

Comment:
Nit: Should we remove these? We already have the patch lines in the other toml file.

Reply:
Can we actually still use these? The intention is to patch only the p22 version. I'm not sure what exactly top-level patches would accomplish TBH.

Reply:
Ah that's a good point... I'm not sure but if this worked for you, maybe we should remove the other ones.

Reply:
Yeah, not sure if top-level patches are still useful or safe, removed for now.